highlevel/robot/controller/motion/trajectory_test.py

```diff
@@ -0,0 +1,3 @@
+"""
+Test the trajectory module.
```

Comment:
is it really tho? :D

Reply:
The tests are left as en exercise to the reader.

Reply:
no!